robottelo/host_helpers/satellite_mixins.py

```diff
@@ -101,6 +106,96 @@ def md5_by_url(self, url):
         return result.stdout
 
 
+class SystemInfo:
+    """Things that needs access to satellite shell for gaining satellite system configuration"""
+
+    @property
+    def get_available_capsule_port(self):
```

Comment:
Generally properties are named like nouns, not like actions. `get_something()` is generally a function call to an instance method, whereas `available_capsule_port` is a *property* of a Satellite instance.  Consider if this property had a `setter` method - does setting `get_available_capsule_port` make sense - or does setting `available_capsule_port` make sense?  I'd say the latter.

Reply:
Resolved by renaming it to `available_capsule_port` !